client/blocks/signup-form/index.jsx

```diff
@@ -776,20 +770,20 @@ class SignupForm extends Component {
 
 	getNotice() {
 		if ( this.props.step && 'invalid' === this.props.step.status ) {
-			return this.globalNotice( this.props.step.errors[ 0 ] );
-		}
-		if ( this.state.notice ) {
-			return this.globalNotice( this.state.notice );
+			return this.globalNotice( this.props.step.errors[ 0 ], 'is-error' );
```

Comment:
I can't seem to test this branch of the code. All the errors I can think of producing (e.g. email already in use, weak password, etc.) are handled in a different way and `this.props.step.errors` is never set.

I think defaulting to `'is-error'` should be safe though.

(The `getNoticeMessageWithLogin` method mentions a possible `2FA_enabled` error which actually appears to have a bug at the moment in production 😬 ... filing a separate issue)

Reply:
It's highly possible that it is no longer reachable - signup and onboarding code is being iterated on very often, and that increases the chance for unreachable branches of code to hang around.